src/plugins/wizard/public/application/components/data_tab/field_search.tsx

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Minor: We can use the shortened license header that Ashwin introduced a while back in #936